torch/csrc/distributed/rpc/tensorpipe_agent.cpp

```diff
@@ -1,3 +1,4 @@
+#ifndef _WIN32
```

Comment:
do we need this ifndef or can just exclude tensorpipe source files when it is windows?

Reply:
Exclude in cmakefile is a better way. @lw has a PR for it, https://github.com/pytorch/pytorch/pull/42682.  In this PR, it will exclude tensorpipe related files with USE_TENSORPIPE. I will rebase my code after it get merged.